packages/server/lib/reporter.coffee

```diff
@@ -132,13 +143,14 @@ reporters = {
 }
 
 class Reporter
-  constructor: (reporterName = "spec", reporterOptions = {}, projectRoot) ->
+  constructor: (reporterName = "spec", reporterOptions = {}, projectRoot, slowTestThreshold) ->
```

Comment:
I don't think slowTestThreshold belongs here. I think it should be passed into the mocha constructor as the `slow` option, which would cut down on the verbosity of adding this option. 

Also we need to whitelist sending the `_slow` prop from the runner to the reporter